go/consensus/tendermint/apps/staking/staking.go

```diff
@@ -120,6 +120,12 @@ func (app *stakingApplication) ExecuteMessage(ctx *api.Context, kind, msg interf
 			return app.transfer(ctx, state, m.Transfer)
 		case m.Withdraw != nil:
 			return app.withdraw(ctx, state, m.Withdraw)
+		case m.AddEscrow != nil:
+			// TODO: probably nicer if `processRuntimeMessages` instead stores into context
```

Comment:
cc @kostko  what do you think?

Reply:
Yeah I had the same thought while looking at the draft ;-)

Reply:
updated